.github/dependabot.yaml

```diff
@@ -13,3 +13,9 @@ updates:
       interval: daily
     commit-message:
       prefix: "chore(deps)"
+  - package-ecosystem: pip
```

Comment:
not sure if we want dependabot updating all of the python stuff, but don't think it should hurt.
If we need to support older versions then we can make a CI run dedicated to the older one.

Reply:
Would this conflict with `poetry.lock`?

Reply:
no, dependabot uses 'pip' for *all* python deps, https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem
![image](https://user-images.githubusercontent.com/101757619/221372551-30d04cd5-86e7-4143-9de7-d63a3c7c26b3.png)